modules/search.py

```diff
@@ -125,22 +121,34 @@ def bing(phenny, input):
 bing.commands = ['bing']
 bing.example = '.bing swhack'
 
-r_duck = re.compile(r'nofollow" class="[^"]+" href="(http.*?)">')
+r_duck = re.compile(r'nofollow" class="[^"]+" href=".+?(http.*?)">')
```

Comment:
I don't exactly know how ddg's search has changed so it's hard to explain why that addition is needed, but here's what it's getting (searching for 'ayy'):
<a rel="nofollow" class="result__a" href="/l/?kh=-1&amp;uddg=http%3A%2F%2Fwww.urbandictionary.com%2Fdefine.php%3Fterm%3DAyy">

I suppose that could be changed with '.*' or '.+', but '.+?' was just the first thing I found to get it to work so I kept it.